data_processing/data_processing.py

```diff
@@ -23,3 +29,186 @@ def data_processing(video_path: Path, output_path: Path) -> List[Path]:
 
 if __name__ == "__main__":
     data_processing(Path(sys.argv[1]), Path(sys.argv[2]))
+
+
+def separate_images(source_dirs, output_dir, binary=False):
```

Comment:
I like how you combined the binary and multiclass into one function with a boolean `binary` parameter.

Reply:
Thanks!